src/main/java/jalau/at18/architects/ScienceCounter.java

```diff
@@ -0,0 +1,32 @@
+package jalau.at18.architects;
+
+import java.util.LinkedList;
+
+public class ScienceCounter {
+    private LinkedList<String> listOfScienceElements;
+
+    public ScienceCounter(LinkedList<String> listOfScienceElements) {
+        this.listOfScienceElements = listOfScienceElements;
+    }
+
+    public LinkedList<String> getListOfScienceElements() {
+        return listOfScienceElements;
+    }
+
+    public void addElementToList(GreenCard greenCard) {
+        listOfScienceElements.add(greenCard.getSymbol());
+    }
+
+    public boolean traverseTheScienceElementsList() {
+        boolean sameValue = (listOfScienceElements.getFirst() == listOfScienceElements.getLast()) ? true : false;
+        return sameValue;
+    }
+
+    public void removeElementFromList() {
+        listOfScienceElements.remove();
+    }
+
+    public int sizeOfScienceELementsList() {
```

Comment:
Good evening Jose, we decided to make it like that because we will restrict the use of the list to only the neccesary methods, this is the reason to create this class, other than that it will be a little bit easy to identify the specific class

Reply:
Ok, I see that this was also renamed and is easier to understand. Thanks for the updates